.github/workflows/testing.yml

```diff
@@ -9,7 +9,6 @@ jobs:
     strategy:
       matrix:
         py:
-          - 3.6
```

Comment:
Rather than remove 3.6 outright, why not change the GH worker instance to be an `ubuntu-20.04`? Given that when run in anger, this most likely will be running on Python 3.6 (default on SLE 15 SP3)?

Reply:
Excellent idea, thanks!